src/pal/src/CMakeLists.txt

```diff
@@ -5,6 +5,11 @@ if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
   set(CLR_CMAKE_USE_SYSTEM_LIBUNWIND 1)
 endif(CMAKE_SYSTEM_NAME STREQUAL Darwin)
 
+if(CMAKE_SYSTEM_NAME STREQUAL FreeBSD)
+  # On FreeBSD , we use the libunwind that's part of the OS
+  set(CLR_CMAKE_USE_SYSTEM_LIBUNWIND 1)
+endif(CMAKE_SYSTEM_NAME STREQUAL FreeBSD)
```

Comment:
Since this is identical to the Darwin block above, could these be combined with an OR condition on the name?

Reply:
+1

Reply:
I was thinking about it @stephentoub but I felt this may be easier to read.
One way would be to change it to if(NOT CMAKE_SYSTEM_NAME STREQUAL Linux)
e.g. use our copy only for Linux and use system version for OSX and all BSD variants.

Reply:
OR is used a bunch of times elsewhere in the file; same for AND.  If `if(NOT CMAKE_SYSTEM_NAME STREQUAL Linux)` makes more sense logically, then I'm fine with it.  I just don't want us having unnecessary duplication.

Reply:
I decided to go with OR. This can be revisited later if we ever have more non-linux platforms.